app/src/common/shared/com/igalia/wolvic/ui/OffscreenDisplay.java

```diff
@@ -124,6 +127,8 @@ public void onCreate(Bundle savedInstanceState) {
             try {
                 getWindow()
                         .getDecorView()
+                        // TODO: These flags and setSystemUiVisibility method have been deprecated
+                        //  https://developer.android.com/reference/android/view/View
```

Comment:
Ditto opening an issue for this

Reply:
https://github.com/Igalia/wolvic/issues/800